proto/message_contents/public_key.proto

```diff
@@ -27,50 +26,58 @@ message UnsignedPublicKey {
     }
 }
 
-// Signed public key that uses self-describing signatures (aka
```

Comment:
I assume these were all added in the previous prototype and so we're not worried about backward compatibility with these?